infra/network-firewall.tf

```diff
@@ -0,0 +1,64 @@
+data "aws_caller_identity" "current" {}
+
+locals {
+  region     = "eu-south-1"
+  account_id = data.aws_caller_identity.current.account_id
+  vpc_cidr   = "10.0.0.0/16"
+  num_azs    = 3
+  azs        = slice(data.aws_availability_zones.available.names, 0, local.num_azs)
```

Comment:
why we need this locals?
- region is already defined in a variabile
- probably account_id is already defined in other local
- vpc_cidr is already defined
- azs is not used
- num_azs probably we can define as variable or more generic local in other

Reply:
changes present in the latest commit